docs/src/_reference/cloud-cli.md

```diff
@@ -25,6 +25,41 @@ Logging out of Meltano Cloud invalidates your login token, and deletes the local
 meltano cloud logout
 ```
 
+## `project`
+
+The `project` command provides an interface for Meltano Cloud projects.
+
+The `list` subcommand shows all of the projects you have access to, and can use with other commands:
+
+```sh
+meltano cloud project list
+╭──────────┬───────────────────────────────┬──────────────────────────────────────────────────────────╮
+│  Active  │ Name                          │ Git Repository                                           │
+├──────────┼───────────────────────────────┼──────────────────────────────────────────────────────────┤
+│          │ Meltano Squared               │ https://github.com/meltano/squared.git                   │
+│          │ MDS-in-a-box                  │ https://github.com/aaronsteers/meltano-demo-in-a-box.git │
+╰──────────┴───────────────────────────────┴──────────────────────────────────────────────────────────╯
+```
+
+When you run the `login` command, if you only have a single project, it will be activated by default. Otherwise, you will need to run `meltano cloud project activate` to specify which Meltano Cloud project the other `meltano cloud` commands should operate on.
+
+```sh
+meltano cloud project activate 'Meltano Squared'
+Activated Meltano Cloud project 'Meltano Squared'
+```
+
+```sh
+meltano cloud project list
+╭──────────┬───────────────────────────────┬──────────────────────────────────────────────────────────╮
+│  Active  │ Name                          │ Git Repository                                           │
+├──────────┼───────────────────────────────┼──────────────────────────────────────────────────────────┤
+│    X     │ Meltano Squared               │ https://github.com/meltano/squared.git                   │
```

Comment:
@WillDaSilva @aaronsteers we'd talked somewhere about having the referenceable name for a project to always be a lower case slug with just a-z and dashes (maybe digits too?) We can still have a pretty name, but keeping the slugified name would make the CLI syntax easier so quotes aren't necessary. 

That way the above command could be `meltano cloud project activate meltano-squared` (or whatever the slug is).

Reply:
@WillDaSilva makes sense to me. Will you make an issue around this so we don't lose track?

Reply:
- https://github.com/meltano/infra/issues/582

With this API change made, no changes should be required on the CLI side. We may want to add a doc update to say that normalization is performed on the name, so e.g. it isn't case sensitive, but this isn't strictly necessary since the existing docs won't become incorrect because of this change.